.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: "Close stale issues"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This has been marked stale after 30 days with no activity. It will be closed in 5 days if there is no activity.'
+        stale-pr-message: 'Stale pull request message'
+        days-before-stale: 30
+        days-before-close: 5
+        stale-issue-label: 'stale'
+        exempt-issue-labels: 'good first issue,help wanted,in progress,on hold,in review'
```

Comment:
Thanks for including this thorough list.

Just a note: the items on our [roadmap](https://github.com/orgs/newrelic/projects/12) are currently represented as issues, and we don't want those to be closed as stale, so if we continue to organize things in that way we'll need to add another exemption label.